src/components/Requirements/IncompleteSelfCheck.vue

```diff
@@ -141,50 +138,14 @@ export default defineComponent({
     },
     addExistingCourse(option: string) {
       this.showDropdown = false;
-      const { uniqueID, crseId } = this.selfCheckCourses[option];
-      updateRequirementChoice(uniqueID, choice => ({
-        ...choice,
-        // Since we edit from a self-check requirement,
-        // we know it must be `acknowledgedCheckerWarningOptIn`.
-        acknowledgedCheckerWarningOptIn: Array.from(
-          new Set([...choice.acknowledgedCheckerWarningOptIn, this.subReqId])
-        ),
-        // Keep existing behavior of keeping it connected to at most one requirement.
-        optOut: getRelatedRequirementIdsForCourseOptOut(
-          crseId,
-          this.subReqId,
-          store.state.groupedRequirementFulfillmentReport,
-          store.state.toggleableRequirementChoices,
-          store.state.userRequirementsMap
-        ),
-      }));
```

Comment:
This should have the same functionality, except refactored. Also, `getRelatedRequirementIdsForCourseOptOut` is no longer necessary, since courses can be connected to multiple requirements that conflict with each other.